### DISCUSSION TITLE
Opacity value with Hex instead of Rgb

### DISCUSSION THREAD

**Discussion by timheerwagen** [tailwindlabs/tailwindcss#7125] [Help]
In the documentation i found following code to use tailwinds color opacity modifiers with css variables:

```
function withOpacityValue(variable) {
  return ({ opacityValue }) => {
    if (opacityValue === undefined) {
      return `rgb(var(${variable}))`
    }
    return `rgb(var(${variable}) / ${opacityValue})`
  }
}

module.exports = {
  theme: {
    colors: {
      primary: withOpacityValue('--color-primary'),
      secondary: withOpacityValue('--color-secondary'),
      // ...
    }
  }
}
```

Is there any way to use this with hex variables instead of rgb?

**Comment by Noitidart** [comment]
Why do you want hex? Is it for older browser support? In which case you should return rgba instead of hex. the rgb/opacity isn't supported in older browser.

**Comment by acristernam** [comment]
@timheerwagen did you come up with a solution? I'm also looking how to do this

**Reply by Noitidart** [reply]
I couldn't find a solution. The tailwind support team says we can use postcss but I and another guy on Stackoverflow couldn't get it to work. - https://github.com/dmarchena/postcss-color-rgb

So what we did is disable opacity, which isn't ideal, I do need opacity on things like focus:ring :( 

Here's the way to disable opacity which gives you rgb - https://stackoverflow.com/a/71324360/1828637
